index.android.js

```diff
@@ -20,7 +20,7 @@ export default class FlowApp extends Component {
   render() {
     return (
       <Navigator
-        initialRoute={{ title: 'Flow', name: 'splash' }}
+        initialRoute={{ title: 'Flow', name: 'meters' }}
```

Comment:
This file shouldn't be committed to dev, since the app needs to have the splash screen come up first. The rest of your changes are on point, though, good work so far! Nearly there it looks like.